sys-libs/gpm/gpm-1.20.7-r3.ebuild

```diff
@@ -0,0 +1,100 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# emacs support disabled due to #99533 #335900
+
+EAPI=7
+
+inherit systemd toolchain-funcs autotools multilib-minimal usr-ldscript
+
+DESCRIPTION="Console-based mouse driver"
+HOMEPAGE="https://www.nico.schottelius.org/software/gpm/"
+SRC_URI="https://www.nico.schottelius.org/software/${PN}/archives/${P}.tar.lzma
+	mirror://gentoo/${P}-docs.patch.xz"
```

Comment:
mirror://gentoo is no valid SRC_URI

Reply:
> mirror://gentoo is no valid SRC_URI

Almost 5 years now since [this commit](https://gitweb.gentoo.org/repo/gentoo.git/commit/sys-libs/gpm/gpm-1.20.7-r2.ebuild?id=2cbf041b6cf2fea23d2a706521ee35b948201174).  And then it suddenly turns invalid?

Reply:
The typical solution to this is to host the file on dev.gentoo.org, but you would need to be a Gentoo developer to do that.